crates/tiny/config.yml

```diff
@@ -40,6 +42,7 @@ servers:
       # sasl:
       #   username: tiny_user
       #   password: hunter2
+      #   pem: "/home/user/.config/tiny/oftc.pem"
```

Comment:
Could you also clarify somewhere (maybe right above the `sasl:` line) that you either need `pem` or `username` + `password`? Currently it looks like you need all of them if you want to use SASL authentication.

Reply:
I tried to make this clear with `Choose only one of the options below.` in the general sasl description at the top. But maybe I should rephrase somehow..

Reply:
Choose only one of the options below. -> Choose either `username` and `password` _or_ `pem` in the section below.